### PR TITLE
All `include_vars` need to have `*.yml`, `*.yaml` or `*.json` extension.

### PR DESCRIPTION
**Pull request by guits**
As introduced in the following PR: https://github.com/ansible/ansible/pull/17207, we need to refactor our code.

### CODE REVIEW

**Comment by alfredodeza** [comment]
I don't see a link to a relevant PR. Could you explain a bit more the *why*?

**Comment by leseb** [comment]
@alfredodeza oops it's the PR 17207 of ansible, @guits is going to update the commit message with the full URL.

**Comment by leseb** [comment]
See also: https://github.com/ansible/ansible/issues/18223
However it seems we can't reproduce this anymore.

**Comment by alfredodeza** [comment]
jenkins test ansible2.2-centos7-cluster

File: README.md [hunk 1]
```diff
@@ -259,7 +259,7 @@ Install the Vagrant plugin for the openstack provider: `vagrant plugin install v
 
 ```bash
 $ cp site.yml.sample site.yml
-$ cp group_vars/all.docker.sample group_vars/all
+$ cp group_vars/all.docker.sample group_vars/all.yml
```
Comment (leseb) [inline README.md]: `group_vars/all.docker.yml.sample`?

**Pull request merged**